File: c/1008.c
#include <stdio.h>

int main() {
    int num, hour;
    double salary_per_hour;

    scanf("%d %d %lf", &num, &hour, &salary_per_hour);
    printf("NUMBER = %d\n", num);
    printf("SALARY = U$ %.2lf\n", hour * salary_per_hour);

    return 0;
}

File: src/javascript/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var num = parseInt(lines.shift());
var hour = parseInt(lines.shift());
var salary_per_hour = parseFloat(lines.shift());

console.log('NUMBER = ' + num);
console.log('SALARY = U$ ' + (hour * salary_per_hour).toFixed(2));